src/mobility/components/CityBikeStationBottomSheet.tsx

```diff
@@ -36,6 +42,7 @@ export const CityBikeStationSheet = ({station, close}: Props) => {
     appStoreUri,
     rentalAppUri,
   });
+  const stationName = getTextForLanguage(station.name.translation, language);
```

Comment:
Just a note: Also possible to use `useTextForLanguage` here if you don't want to provide `language`.

Reply:
Thank you, I wasn't aware of that one.